spec/tap_installer_spec.rb

```diff
@@ -3,8 +3,12 @@
 require "spec_helper"
 
 describe Bundle::TapInstaller do
-  def do_install(clone_target = nil)
-    Bundle::TapInstaller.install("phinze/cask", clone_target)
+  def do_install(options = {})
+    Bundle::TapInstaller.install("phinze/cask", options)
```

Comment:
Can we change `phinze/cask` to something else here?

Reply:
`homebrew/cask` I guess, thanks.